src/matcha_ml/state/remote_state_manager.py

```diff
@@ -19,6 +21,8 @@
 from matcha_ml.templates.run_state_storage_template import TemplateRunner
 
 DEFAULT_CONFIG_NAME = "matcha.config.json"
+LOCK_FILE_NAME = "matcha.lock"
+ALREADY_LOCKED_MESSAGE = "Remote state is already locked."
```

Comment:
Should we direct users to run `matcha force-unlock` command to force unlock the remote state?

Reply:
Good point, I was actually meaning to improve this message